src/Parser/Hunter/Survival/Modules/Talents/ButcheryCarve.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+import SpellIcon from 'common/SpellIcon';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellLink from 'common/SpellLink';
+import ItemDamageDone from 'Main/ItemDamageDone';
+import StatisticBox from 'Main/StatisticBox';
+import ITEMS from 'common/ITEMS/HUNTER';
+import Wrapper from 'common/Wrapper';
+
+class ButcheryCarve extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bonusDamage = 0;
+  damageHits = 0;
+  casts = 0;
+  totalStacksUsed = 0;
+
+  get averageTargetsHit() {
+    return (this.damageHits / this.casts).toFixed(2);
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.BUTCHERY_TALENT.id && spellId !== SPELLS.CARVE.id) {
+      return;
+    }
+    if (this.combatants.selected.hasBuff(SPELLS.BUTCHERS_BONE_APRON_BUFF.id, event.timestamp)) {
+      this.totalStacksUsed += this.combatants.selected.owner.modules.butchersBoneApron.apronStacks;
+    }
+    this.casts++;
+  }
+
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.BUTCHERY_TALENT.id && spellId !== SPELLS.CARVE.id) {
+      return;
+    }
+    this.damageHits++;
+    this.bonusDamage += event.amount + (event.absorbed || 0);
+  }
+
+  get averageTargetsThreshold() {
+    return {
+      actual: this.averageTargetsHit,
+      isLessThan: {
+        minor: 2,
+        average: 1.75,
+        major: 1.5,
+      },
+      style: 'number',
+    };
+  }
+
+  suggestions(when) {
+    const spellLink = this.combatants.selected.hasTalent(SPELLS.BUTCHERY_TALENT.id) ? SPELLS.BUTCHERY_TALENT : SPELLS.CARVE;
+    when(this.averageTargetsThreshold).addSuggestion((suggest, actual, recommended) => {
```

Comment:
Assuming butcher's bone apron makes it worth casting butchery/carve single target, I think it would be a good idea to use another threshold when it's equipped.

Reply:
Added a note in the suggestion to clarify possible usage if using Butchers Bone Apron on ST, but recommending to try and use something else.